public/app/plugins/panel/timeseries/TimeSeriesPanel.tsx

```diff
@@ -27,7 +29,17 @@ export const TimeSeriesPanel: React.FC<TimeSeriesPanelProps> = ({
     return getFieldLinksForExplore({ field, rowIndex, range: timeRange });
   };
 
-  const { frames, warn } = useMemo(() => prepareGraphableFields(data?.series, config.theme2), [data]);
+  const { semanticFields } = options;
+
+  const { frames, warn } = useMemo(() => {
+    const prepped = prepareGraphableFields(data?.series, config.theme2);
+
+    if (prepped.frames && semanticFields) {
```

Comment:
For OHLC chart, should we require the semanticFields option to be completely configured? Guess having only i.e. open configured doesn't make much sense.